<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioPanel</class>
 <widget class="QWidget" name="AudioPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>123</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="audioPreviewCheck">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Baseband audio preview</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Demodulator</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="demodCombo">
        <item>
         <property name="text">
          <string>AM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LSB</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Sample rate</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="sampleRateCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Audio cutoff</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="cutoffSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="cutoffLabel">
        <property name="minimumSize">
         <size>
          <width>62</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>22222 Hz</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Volume</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="volumeSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="volumeLabel">
        <property name="text">
         <string>0%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
